File: WineryAPI/Models/Bure.cs
using System;
using System.Collections.Generic;

namespace WineryAPI.Models;

public partial class Bure
{
    public int Idbur { get; set; }

    public decimal Zapremina { get; set; }

    public string Materijal { get; set; } = null!;

    public string Oznakabur { get; set; } = null!;

    public int? PodrumIdpod { get; set; }

    public virtual Podrum? PodrumIdpodNavigation { get; set; }

    public virtual ICollection<SeLageruje> SeLagerujes { get; set; } = new List<SeLageruje>();
}

File: winery-frontend/src/types/bure.ts
export interface Bure {
  idbur: number
  zapremina: number
  materijal: string
  oznakabur: string
  podrumIdpod: number | null
  podrumNaziv: string | null
}

export interface BureDetail {
  idbur: number
  zapremina: number
  materijal: string
  oznakabur: string
  podrumIdpod: number | null
  podrumNaziv: string | null
  brojLagerovanihVina: number
}

export interface CreateBureRequest {
  zapremina: number
  materijal: string
  oznakabur: string
  podrumIdpod: number
}

export interface UpdateBureRequest {
  zapremina: number
  materijal: string
  oznakabur: string
}

